src/basic/utils/createElement.js

```
@@ -0,0 +1,9 @@
+const createElement = ({ elementType, className, innerHtml, ...props }) => {
```

ESLint

[error] 'props' is defined but never used.

(no-unused-vars)